submissions/ArthurGorbenko/html.css.practice/README.md

```diff
@@ -0,0 +1,2 @@
+## HTML CSS POPUP
+[demo](https://arthurgorbenko.github.io/)
```

Comment:
For this task, probably, you would better use Project Site instead of User Site
http://i.imgur.com/rs2BLQr.png

Have a look for the difference in my example:
* User Site https://aymkin.github.io/
* Project Site https://aymkin.github.io/coogle-style/#

Reply:
Yep, that's more practical, thanks.

Reply:
I guess,something like [this](https://github.com/kottans/frontend-2019-p2p/pull/67/commits/a9cd8a9edcddefb526cb8c93c6156819a45dd284).